src/renderer/lib/hooks/useToasts.ts

```diff
@@ -0,0 +1,10 @@
+import { useSnackbar } from "notistack";
+
+export const useToasts = () => {
+  const { enqueueSnackbar } = useSnackbar();
+  return {
+    showError: (message: string) => enqueueSnackbar(message, { variant: "error" }),
```

Comment:
```suggestion
    showError: (message: string, options?: OptionsObject) => enqueueSnackbar(message, { ...options, variant: "error" }),
```
in case we want to pass more options from where ever these get called

OptionsObject comes from notistack

Reply:
I considered this too. However I think for the time being having one consistent interface for emitting notifs means that all the notifs will exude the same consistent behaviour, rather than individual notifs behaving different. It also means that if we ever change toast libraries again, we won't have to modify all the individual files that emit toasts since if they pass in an `options` object of type `OptionsObject`, they would need to also be updated to whatever the new library supports.